README.md

```diff
@@ -1,32 +1,22 @@
 # gRPC for .NET
 
-## Preview out!
+## Available now on .NET Core 3.0!
 
-A preview of gRPC for ASP.NET Core is on [NuGet](https://www.nuget.org/packages/Grpc.AspNetCore.Server). A template using the preview [shipped with .NET Core 3.0 Preview 3](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-core-3-0-preview-3/).
+gRPC is a modern, open source, high-performance remote procedure call (RPC) framework that can run anywhere. gRPC enables client and server applications to communicate transparently, and simplifies the building of connected systems.
 
-See https://github.com/grpc/grpc for the official version of gRPC C# (ready for production workloads).
+gRPC functionality for .NET Core 3.0 includes:
 
-## The plan
+* [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCore) &ndash; An ASP.NET Core framework for hosting gRPC services. gRPC on ASP.NET Core integrates with standard ASP.NET Core features like logging, dependency injection (DI), authentication and authorization.
+* [Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client) &ndash; A gRPC client for .NET Core that builds upon the familiar `HttpClient`.
+* [Grpc.Net.ClientFactory](https://www.nuget.org/packages/Grpc.Net.ClientFactory) &ndash; gRPC client integration with `HttpClientFactory`.
```

Comment:
nit: It might not be immediately obvious to everyone what's the value of integration of with `HttpClientFactory`. Optionally you can add a sentence that this simplifies the common microservices scenario where server-side handler invoke other services (as clients)?